docs/changelog/CHANGELOG.md

```diff
@@ -18,8 +18,13 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 
 ## tip
 
+**Update note 1: changed a meaning of `-maxLabelsPerTimeseries`, `-maxLabelNameLen` and `-maxLabelValueLen`. Previously excessive labels, label names and values were truncated. With this change all series that are hitting the limits will be ignored instead.**
+
 * SECURITY: upgrade Go builder from Go1.23.3 to Go1.23.4. See the list of issues addressed in [Go1.23.4](https://github.com/golang/go/issues?q=milestone%3AGo1.23.4+label%3ACherryPickApproved).
 
+* FEATURE: [vmagent](https://docs.victoriametrics.com/vmagent): support `-maxLabelsPerTimeseries`, `-maxLabelNameLen` and `-maxLabelValueLen` flags to limit amount of labels, label name and value length for pushed or scraped series. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6928).
+* FEATURE: [vmsingle](https://docs.victoriametrics.com/single-server-victoriametrics/) changed a meaning of `-maxLabelsPerTimeseries`, `-maxLabelNameLen` and `-maxLabelValueLen`. Previously excessive labels, label names and values were truncated. With this change all series, which are hitting these limits will be ignored. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6928)
```

Comment:
`maxLabelNameLen` is a new flag. Shall we mention it explicitly in the note?

Reply:
updated